Manager/RedirectRouteNotUniqueException.php

```diff
@@ -24,9 +24,9 @@ class RedirectRouteNotUniqueException extends \Exception
     /**
      * @param string $source
```

Comment:
```suggestion
     * @param string $source
     * @param string|null $sourceHost
```

Reply:
Hi,

No problem at all; we've forked this repo. But it would be neat if the change could make it into the official repo, so we have a bit less maintenance overhead in the future. On the other hand it's not a big problem if you would decide not to merge it after all.

In any case, thanks a lot for taking the time to do the PR review! Your comments have been processed and the branch has been rebased off `0.x`.

Please let me know if you spot any other improvements. Thanks again!